qa/util/src/main/java/io/camunda/qa/util/cluster/TestOperateWithExporter.java

```diff
@@ -148,7 +148,6 @@ protected SpringApplicationBuilder createSpringBuilder() {
 
     withProperty("zeebe.broker.gateway.enable", true);
     withProperty("camunda.rest.enabled", true);
```

Comment:
🤔 [not in the scope] Should we consider enable this property as true by default in the future?

Reply:
It already is, actually, see [default config test](https://github.com/camunda/camunda/blob/88d0b4df0681c15923aa5c51087756bb3e6e00dd/zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/configuration/RestApiDefaultConfigurationTest.java#L35-L47)